src/iexamine.cpp

```diff
@@ -3189,6 +3240,12 @@ void iexamine::water_source( player &p, const tripoint &examp )
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::clean_water_source( player &, const tripoint &examp )
+{
+    item water = item( "water_clean", 0, item::INFINITE_CHARGES );
+    g->handle_liquid( water, nullptr, 0, &examp );
```

Comment:
Ultimately, I dont understand why the build is failing with
`src/iexamine.cpp:3197:8: error: ‘class game’ has no member named ‘handle_liquid’` especially since the handle_liquid is used in the same manner a few lines above

Even worse, the changes compile on my windows machine, and it also works properly in-game. Any hints of what could be the issue?